editor/components/block-list/block.js

```diff
@@ -50,6 +51,7 @@ import {
 	isBlockMultiSelected,
 	isBlockSelected,
 	isFirstMultiSelectedBlock,
+	isSelectionDisable,
```

Comment:
Here and across the diff, should read _isSelectionDisabled_ instead. The term _toggleSelection_ is fine, however.